src/NewValueHolder.package/NewValueHolder.class/properties.json

```diff
@@ -1,6 +1,6 @@
 {
 	"commentStamp" : "PeterUhnak 3/5/2016 11:16",
-	"super" : "Object",
```

Comment:
Why is NewValueHolder now subclass of Model?

Reply:
May be this is the wrong analysis from me.
In fact PluggableButtonMorph requires something that can accept addDependent:
So when I converted the Value to NewValueHolder I found this.

Reply:
yes, and Model is the right place to put that stuff... not object (we need to remove that mechanism from Object, btw... but is hard :P)